### PR TITLE
all close

### PR DESCRIPTION
**Pull request by rijobro**
### Description
allclose function for testing

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [x] Quick tests passed locally by running `./runtests.sh --quick --unittests`.

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -55,27 +56,23 @@ def clone(data: NdarrayTensor) -> NdarrayTensor:
...
-    if isinstance(a, np.ndarray) and isinstance(b, np.ndarray):
-        return np.allclose(a, b)
-
-    if isinstance(a, torch.Tensor) and isinstance(b, torch.Tensor):
-        return torch.allclose(a, b)
-
-    return False
+    a = a.cpu() if isinstance(a, torch.Tensor) else a
+    b = b.cpu() if isinstance(b, torch.Tensor) else b
+    return np.allclose(a, b, rtol, atol, equal_nan)
```
Comment (wyli) [inline tests/utils.py:75]: `np.testing.assert_allclose()` will print more info:
```python
>>> np.allclose([1e10,1e-7], [1.00001e10,1e-8])
False
>>> np.testing.assert_allclose([1e10,1e-7], [1.00001e10,1e-8])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/anaconda3/envs/py37/lib/python3.7/site-packages/numpy/testing/_private/utils.py", line 1531, in assert_allclose
    verbose=verbose, header=header, equal_nan=equal_nan)
  File "/usr/local/anaconda3/envs/py37/lib/python3.7/site-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

Mismatched elements: 2 / 2 (100%)
Max absolute difference: 100000.
Max relative difference: 9.
 x: array([1.e+10, 1.e-07])
 y: array([1.00001e+10, 1.00000e-08])
```

Comment (rijobro) [reply tests/utils.py]: Thanks, updated to use `np.testing.assert_allclose`.

**Review by wyli — APPROVED** [review]
thanks!

**Comment by rijobro** [comment]
Clicking on the blossom-ci error, it takes me to the results for the [Torch flip PR](https://github.com/Project-MONAI/MONAI/actions/runs/1163257210).

**Pull request merged**